src/pages/networks/forms/NetworkFormDns.tsx

```diff
@@ -13,13 +13,17 @@ const NetworkFormDns: FC<Props> = ({ formik }) => {
   return (
     <ScrollableConfigurationTable
       rows={[
-        getConfigurationRow({
-          formik,
-          name: "dns_domain",
-          label: "DNS domain",
-          defaultValue: "",
-          children: <Input type="text" />,
-        }),
+        ...(formik.values.networkType !== "physical"
```

Comment:
Maybe it's a little clear to define the configs and assign them to a variable?
```
const networkConfigsDNS = formik.values.networkType === "bridge"
    ? [
        getConfigurationRow({
            formik,
            name: "dns_domain",
            label: "DNS domain",
            defaultValue: "",
            children: <Input type="text" />,
        }),
        getConfigurationRow({
            formik,
            name: "dns_mode",
            label: "DNS mode",
            defaultValue: "",
            children: (
                <Select
                options={[
                    {
                    label: "Select option",
                    value: "",
                    disabled: true,
                    },
                    {
                    label: "None",
                    value: "none",
                    },
                    {
                    label: "Managed",
                    value: "managed",
                    },
                    {
                    label: "Dynamic",
                    value: "dynamic",
                    },
                ]}
                />
            ),
        }),
        getConfigurationRow({
            formik,
            name: "dns_search",
            label: "DNS search",
            defaultValue: "",
            children: <Textarea />,
        }),
    ] : [
        getConfigurationRow({
            formik,
            name: "dns_nameservers",
            label: "DNS nameservers",
            defaultValue: "",
            children: <Input type="text" />,
        }),
    ]
```

Reply:
Apart from the row configurations, there is nothing more in this component, so I'd keep it straight forward and not introduce an indirection with a variable.